handlers/shake.py

```diff
@@ -491,6 +491,11 @@ def get(self, shake_name=None):
         if sharedfiles:
             build_date = sharedfiles[0].feed_date()
 
+        # If this is the "mltshp" RSS feed, use original files so that links
+        # won't go to the seemingly "unpopular" copy.
+        if shake_name == "mltshp":
```

Comment:
Should compare against `options.best_of_user_name` instead of hardcoding "mltshp".

Reply:
Sure thing.